poetry.lock

```diff
@@ -20,18 +31,18 @@ tests-no-zope = ["cloudpickle", "hypothesis", "mypy (>=1.1.1)", "pympler", "pyte
 
 [[package]]
 name = "bandit"
-version = "1.7.5"
+version = "1.7.6"
 description = "Security oriented static analyser for python code."
 optional = false
-python-versions = ">=3.7"
+python-versions = ">=3.8"
```

Comment:
same question here

Reply:
I don't have any control over this, which seems to be coming from this dependency's new version.